arcade/sound.py

```diff
@@ -52,11 +54,11 @@ def play(self, volume=1.0, pan=0.0):
         if not _audiolib:
             return
 
-        _audiolib.play(self.wav_file,
-                       aVolume=volume,
-                       aPan=pan,
-                       aPaused=0,
-                       aBus=0)
+        self.voice_handle = _audiolib.play(self.wav_file,
```

Comment:
The volume/pan controls (while the file is playing) requires a voice handle.  This is different from the setVolume/getVolume that uses the object handle (which seems to set the default volume of the NEXT sound to be created).